core/src/main/java/org/apache/iceberg/util/DateTimeUtil.java

```diff
@@ -88,4 +88,40 @@ public static long microsFromTimestamptz(OffsetDateTime dateTime) {
   public static String formatTimestampMillis(long millis) {
     return DATE_FORMAT.format(LocalDateTime.ofInstant(Instant.ofEpochMilli(millis), ZoneOffset.UTC));
   }
+
+  public static String daysToIsoDate(int days) {
```

Comment:
@rzhang10, please refactor the literals to use these methods where possible. We don't want two separate implementations.

Reply:
Discussed offline since Literals is in API module we are not able to do this refactor.